src/routes/(console)/project-[project]/messaging/providers/table.svelte

```diff
@@ -75,8 +78,12 @@
     <TableBody>
         {#each data.providers.providers as provider (provider.$id)}
             <TableRowLink
-                href={`${base}/project-${$project.$id}/messaging/providers/provider-${provider.$id}`}>
-                <TableCellCheck bind:selectedIds id={provider.$id} />
+                href={$canWriteProviders
```

Comment:
lets try and see if we can use `<svelte:component />` here

Reply:
We tried. didn't work.